lib/gears/object.lua

```diff
@@ -56,6 +56,11 @@ function object:connect_signal(name, func)
     sig.strong[func] = true
 end
 
+-- Register a global signal receiver.
+function object:_connect_everything(callback)
+    table.insert(self._global_receivers, callback)
+end
```

Comment:
It would be cool, but this would require to add the code to `emit_signal`[1]. That's already one of the "hottest" code path in Awesome. Wildcard parsing, even with a compiled state machine, is very slow compared to doing nothing at all. That function is called up to a thousand time per event loop. Also, this isn't really related to this patch.

[1] There is no "list of signals", every object can emit anything.